lisa/transformers/dom0_kernel_installer.py

```diff
@@ -67,8 +67,11 @@ def _output_names(self) -> List[str]:
         return []
 
     def validate(self) -> None:
-        # nothing to validate before source installer started.
-        ...
+        if not isinstance(self._node.os, CBLMariner):
+            raise UnsupportedDistroException(
```

Comment:
Is it a regression, or only work with Azure Linux before?

Reply:
This kernel installer only works with Azure Linux.